fe1-web/parts/lao/organizer/RollCallOpened.tsx

```diff
@@ -29,20 +30,23 @@ const RollCallOpened = () => {
   const route = useRoute();
   const { rollCallID, time } = route.params;
   const navigation = useNavigation();
-  const [, setQrWasScanned] = useState(false);
-  const [attendeesSet, updateAttendeesSet] = useState(new Set<string>());
-  const attendees = Array.from(attendeesSet);
+  const [attendees, updateAttendees] = useState<string[]>([]);
```

Comment:
Why did we switch to a list ? Isn't it problematic if someone is scanned twice ?

Reply:
I use a list here because later we need a list of attendees for requestCloseRollCall, I verify newly scanned data is not included in the attendees' list in handleScan to make sure there will be no duplicate.

Reply:
Okay, it seems reasonable

Reply:
I won't bother you with this given the application, but keep in mind:
1. Effectively, this means you're artificially forcing an array to behave as a set - it might not be super intuitive when reading the code
2. You could always convert from a set to an array later
3. From an algorithmic perspective, this changed the creation of a list of attendees from a complexity of O(n*log(n)) to O(n^2) algorithm

Again, these are no issues in this context, but the change is a bit counter-intuitive to me :) 
At the opposite end of this solution, you could also imagine changing the `requestCloseRollCall` to take a set.

Reply:
Thanks for the comment, I didn't think about the complexity. I will change it back to a set and convert the set to an array for requestCloseRollCall, I won't change requestCloseRollCall for this PR because this will also change JSON.